src/test/resources/smartHouseFeatures/Login.feature

```diff
@@ -0,0 +1,6 @@
+# Login feature review the behavior in thie feature
+Feature: SHLogin
+
+  Scenario: Home page is displayed once set credential in login page
+    Given 'SMARTHOUSE' page is loaded
+    And set admin credentials on 'Login' page
```

Comment:
are you using this file in your test case? if not, please dont upload this file.